# What: Add rasscf as a dependency to mcpdft
# Authors: B. Hajgato (UGent), 2021
#          maxim-masterov (SURF), 2023
#
diff -Nru OpenMolcas-v23.06.orig/CMakeLists.txt OpenMolcas-v23.06/CMakeLists.txt
--- OpenMolcas-v23.06.orig/CMakeLists.txt	2023-10-09 13:47:31.989388000 +0200
+++ OpenMolcas-v23.06/CMakeLists.txt	2023-10-09 13:47:56.971468000 +0200
@@ -2354,6 +2354,9 @@
 
 # set program dependencies of super modules
 # (note dependencies are not recursive with object libraries, so child dependencies must be explicit)
+set (mcpdft_deplibs
+  rasscf
+)
 set (casvb_deplibs
   rasscf
 )
